doc/en/parametrize.rst

```diff
@@ -81,17 +81,27 @@ them in turn:
 
 .. note::
 
-    pytest by default escapes any non-ascii characters used in unicode strings
-    for the parametrization because it has several downsides.
-    If however you would like to use unicode strings in parametrization and see them in the terminal as is (non-escaped), use this option in your ``pytest.ini``:
+    1. Parameter values are passed as-is to tests, and if you mutate them,
+       the mutations will be reflected in subsequent test case calls.
 
-    .. code-block:: ini
+       Specifically, pytest doesn't do any copying of the values, since that would
```

Comment:
I would omit this paragraph. I don't think the user really *needs* to know the rationale, they just need to be aware of it, and I'm inclined to keep explanations brief when possible.

WDYT?

Reply:
Done with new commit.